test/suite/hook_tiered.py

```diff
@@ -52,35 +60,51 @@
 
 # These are the hook functions that are run when particular APIs are called.
 
+# Add the local storage extension whenever we call wiredtiger_open
+def wiredtiger_open_tiered(ignored_self, args):
+    auth_token = "test_token"
+    bucket = "mybucket"
+    extension_name = "local_store"
+    prefix = "pfx-"
+    # XXX This assumes we are running from WT_TEST/test* in the build directory!
+    extension_library = os.getcwd() + \
+            '/../../ext/storage_sources/local_store/.libs/libwiredtiger_local_store.so'
+
+    if not os.path.exists(bucket):
+        os.mkdir(bucket)
+    tier_string = ',,,tiered_storage=(auth_token=%s,' % auth_token + \
```

Comment:
What is the purpose of the three commas `,,,` at the beginning and end of the string?

Reply:
TBH, I just copied that usage from Don's example in `hook_demo.py`.  So I'm not sure.

Reply:
and TBH, I don't remember.  I don't think it has a functional purpose.  Perhaps in testing it was easy to spot.
Try removing it?

Reply:
I removed the extra commas and there were no ill effects.